templates/security/cis.html

```diff
@@ -0,0 +1,121 @@
+      <a href="/support/get-in-touch" class="p-button--positive js-invoke-modal">Access the tooling for audit and automated CIS compliance</a>
+    </div>
+    <div class="col-4 u-vertically-center u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/f98af83d-cis-logo-removebg-preview.png",
+        alt="",
+        width="300",
+        height="300",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-deep">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3>Harden your workloads</h3>
+      <p>Hardening always involves a tradeoff with usability and performance. The default configuration of Ubuntu LTS releases, as provided by Canonical, balances between usability, performance and security. However, systems with a dedicated workload are well-positioned to benefit from hardening. Reduce your workload’s attack surface with Ubuntu and CIS.</p>
+      <br />
+      <a href="/blog/cis-hardened-ubuntu-cyber-attack-and-malware-prevention-for-mission-critical-systems" class="p-button--neutral">Read more about Ubuntu CIS</a>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3>Automate your compliance</h3>
+      <p>Applying a baseline with a large set of instructions manually is not only time consuming but also error-prone. According to <a class="p-link--external" href="https://enterprise.verizon.com/resources/reports/dbir/">Verizon data breach investigations</a> report for 2020, misconfigurations were among the top five reasons for data breaches. Avoid misconfigurations with tooling provided by Canonical that automates your CIS compliance.</p>
+      <br />
+      <a href="https://security-certs.docs.ubuntu.com/en/cis" class="p-button--neutral p-link--external">Read more about Ubuntu’s tooling</a>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3>Audit with OpenSCAP</h3>
+      <p>An important aspect of secure asset configuration for compliance is monitoring. That is being able to verify that systems comply with the selected baseline and contain operating system software supported by the vendor. Ubuntu Pro and Ubuntu Advantage makes content available to audit and monitor systems with the OpenSCAP tool.</p>
+      <br />
+      <a href="https://security-certs.docs.ubuntu.com/en/cis" class="p-button--neutral p-link--external">Read more about Ubuntu’s audit tooling</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h2>Which versions of Ubuntu have tooling for CIS benchmarks?</h2>
```

Comment:
Let's make it shorter:
"Which versions of Ubuntu have CIS tooling?"

I'll update the copydoc

Reply:
I can change it this time, but just for future reference we don't normally do copy updates on the review stage of the process. The copy should be finalised before we start building.

Reply:
I understand but to avoid a misunderstanding in that case I specifically asked for a preview when the pages are ready because (a) these need to be approved and (b) there is no way to know how the web pages will exactly look like when making a google doc. In this particular case the sentence looked too long and out of place in the page.